app/gui/src/model/module/synchronized.rs

```diff
@@ -647,18 +656,18 @@ pub mod test {
                 let actual_old = this.current_ls_version.get();
                 let actual_new =
                     Sha3_224::from_parts(new_content.iter_chunks(..).map(|s| s.as_bytes()));
-                debug!("Actual digest:   {actual_old} => {actual_new}");
-                debug!("Declared digest: {} => {}", edits.old_version, edits.new_version);
-                debug!("New content:\n===\n{new_content}\n===");
+                println!("Actual digest:   {actual_old} => {actual_new}");
+                println!("Declared digest: {} => {}", edits.old_version, edits.new_version);
+                println!("New content:\n===\n{new_content}\n===");
```

Comment:
nit: I believe `debug!` (and other) messages are still visible in regular tests, so this change should be unnecessary.